manifests/m/Microsoft/EdgeWebView2Runtime/95.0.1020.53/Microsoft.EdgeWebView2Runtime.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.EdgeWebView2Runtime
+PackageVersion: 95.0.1020.53
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /install
+  SilentWithProgress: /install
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/d7b45b2a-dbd0-4b36-992f-e9fe03bd548d/MicrosoftEdgeWebView2RuntimeInstallerX64.exe
+  InstallerSha256: EB798BA528A93D0278FEACDB201BD0C365F1948B5CCB915954E012B4D3E3D37C
```

Comment:
I'm not 100% sure, but i think there was a reason why i/we used the `Evergreen Bootstrapper` instead of the `Evergreen Standalone Installer` (which is what we should use yes)
If i'm just being dumb we should probably add the x86 and arm64

Commit after the validation completes to check if it actually works

```suggestion
Installers:
- Architecture: x64
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/d7b45b2a-dbd0-4b36-992f-e9fe03bd548d/MicrosoftEdgeWebView2RuntimeInstallerX64.exe
  InstallerSha256: EB798BA528A93D0278FEACDB201BD0C365F1948B5CCB915954E012B4D3E3D37C
- Architecture: x86
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/72737eb5-9f33-4017-b5d2-12d04a770855/MicrosoftEdgeWebView2RuntimeInstallerX86.exe
  InstallerSha256: 1020F62E5A1DD70C299A0A84849A87EFE9C0A194F271D0484DBBA49E310B5F8C
- Architecture: arm64
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/4a346883-aba4-47d8-906a-f65d27c0eb62/MicrosoftEdgeWebView2RuntimeInstallerARM64.exe
  InstallerSha256: 43C392D825A523397CCC29D4E66011C772F56930CC773E3452F49727338AFE53
```

Reply:
The exes (I *think*) only install for the user, not for the system. That's how it works for Edge, at least.

Reply:
Funny enough, this seems to be the issue
![image](https://user-images.githubusercontent.com/15158490/142024067-1287b890-506c-4ebe-b669-b4d709620640.png)

Reply:
I just found out you need to do `/silent /install`, `/install /silent` did not work...
However it does not prompt for administrator and throws error 2147747609

Reference and possibly another issue on their side? https://github.com/MicrosoftEdge/WebView2Feedback/issues/1349

Reply:
So the main installer exits quickly and therefore throws up the error. ( Correct me if I am wrong) So do you want me to close the pull request?

Reply:
No i don’t think we’re seeing that issue atm.

The error i got was when running winget as non administrator (second issue mentioned)

AFAIK it works correctly when winget is run as Administartor, however it will cause issues since the installer does not promt for Admin. IIRC we have a switch for requires elevated in version 1.1 (next version eta soonMSTM)